File: jour_4/jour4.c
#include<stdio.h>
int main(){
    int a, b, temp;
    printf("Entrez la valeur de a : ");
    scanf("%d", &a);
    printf("Entrez la valeur de b : ");
    scanf("%d", &b);
    temp = a;
    a = b;
    b = temp;
    printf("la valeur de a est : %d\nla valeur de b est : %d\n", a, b);
    /*Autre methode 
    a = a^b
    b = a^b 
    a = a^b
    printf("la valeur de a est : %d\nla valeur de b est : %d\n", a, b);*/
    return 0;
}

File: jour_4/jour4.pas
program ProgramName;
var 
    a, b, temp : integer;
begin
  write('Entrez la valeur a : ');
  readln(a);
  write('Entrez la valeur b : ');
  readln(b);
  temp := a;
  a := b;
  b := temp;
  writeln('la valeur de a est : ', a);
  write('la valeur de b est : ', b);
  //Autre methode
  //a = a^b
  //b = a^b 
  //a = a^b
  //writeln('la valeur de a est : ', a);
  //write('la valeur de b est : ', b);*/
end.